src/components/CodeEditor/TestResultViewer.tsx

```diff
@@ -57,13 +57,16 @@ const TestResultTable = ({ editorOutput }) => {
       </div>
       <div className="h-48 overflow-auto">
         <table className="m-0 min-w-full table-auto border dark:border-slate-800/60">
-          <thead className="bg-gray-100 dark:bg-slate-800/60">
-            <tr>
+          <thead className="bg-white sticky top-0 dark:bg-slate-800/60">
+            <tr >
               {resultTableColumns.map((column) => (
                 <th
                   key={column.key}
                   scope="col"
-                  className="border border-gray-200 p-2 text-center text-xs text-slate-600 dark:border-slate-600/80 dark:text-slate-400"
+                  className=" border-gray-200 p-2 text-center text-xs text-slate-600 dark:border-slate-600/80 dark:text-slate-400 after:content-[''] after:absolute  after:top-0 after:left-0  after:w-full after:h-8 after:border after:border-gray-200 
```

Comment:
The formatting of all these class names needs to be corrected. The ordering of those class names needs to be appropriately sorted. We use the tailwind-prettier plugin to do the sorting automatically in the codebase. Your vscode is not probably setup properly to work with that.

Please have a look into this doc to setup the tailwind-prettier plugin in your machine: https://tailwindcss.com/blog/automatic-class-sorting-with-prettier

Reply:
Thanks for your observation, tailwind prettier setup done.